web/src/test/kotlin/no/nav/su/se/bakover/web/routes/søknad/SøknadRoutesKtTest.kt

```diff
@@ -182,6 +181,7 @@ internal class SøknadRoutesKtTest {
         val personOppslag: PersonOppslag = mock {
             on { person(any()) } doReturn PersonOppslagStub.person(fnr)
             on { aktørId(any()) } doReturn PersonOppslagStub.aktørId(fnr)
+            on { sjekkTilgangTilPerson(any()) } doReturn PersonOppslagStub.sjekkTilgangTilPerson(fnr)
```

Comment:
Savner en tilhørende verify. Savner også en inOrder-blokk og verifyNoMoreInteractions etter inOrder{}-blokka

Vi har vel ikke vært helt konsekvent på dette i testene

Reply:
Det er nok sant, jeg kikket uansett ikke så nøye på disse testene (trengte bare å stubbe det nye kallet for å få de til å funke igjen). 
Når det er sagt så tenker jeg generelt sett at en god del tester fint kan bare stubbe kall til avhengigheter, der de kallene egentlig ikke er en del av det som testes. Men der kallene til avhengighetene er en viktig del av flyten så er det nyttig med mer nøyaktig verifikasjon.